.github/workflows/update-vrt.yml

```diff
@@ -4,8 +4,8 @@ on:
     types: [ created ]
 
 permissions:
-  pull-requests: write
-  contents: write
+  pull-requests: read
```

Comment:
is this because we don't need write permissions?

Reply:
From reading the docs, this is not needed for pushing. But I may be wrong